### PR TITLE
Fix the documentation of HostFactory#create_tokens

### PR DESCRIPTION
**Pull request by kgilpin**

### CODE REVIEW

File: src/host_factory.tokens.create.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Generate multiple tokens at once with the `count` parameter. By default, one tok
 |404|Host factory not found|
 
 + Parameters
-    + id: redis%2Fdefault (string) - ID of the host factory, query-escaped
+    + id: redis/default (string) - ID of the host factory, query-escaped
```
Comment (dividedmind) [inline src/host_factory.tokens.create.md]: I'm confused. Wasn't that the whole change?

Comment (ghost) [reply src/host_factory.tokens.create.md]: i think the change is marking expiration as not optional and CIDR as optional.

ID needs to be `redis/default` or else the HF API tests don't pass.

**Comment by kgilpin** [comment]
We also need to show the escaping of the parameter

**Comment by kgilpin** [comment]
@dustinmm80 where did we end up on this? My suggestion was to change the HF name from `redis/default` to `redis` to avoid the escaping issue altogether, since we are showing an incorrect example right now and there's no obvious way to use a correct one without breaking the test harness.

**Comment by ghost** [comment]
Your suggestion sounds good Kevin, can you update the string? Once tests pass I will merge.

**Comment by ghost** [comment]
I made this change, tests are passing.

https://jenkins.conjur.net/job/conjurinc--apidocs/job/fix-host-factory-create-tokens/20/

Merging!

**Pull request merged**